### ISSUE TITLE
Feature request: tap tab title to refresh

### ISSUE THREAD

**Issue by Linecutterx** [Docile-Alligator/Infinity-For-Reddit#381]
I use multi-Reddits a lot & often sort by new (!). It would be easier to access refresh if it were triggered by tapping the tab title, which I can bring up with a short reverse scroll.

**Comment by matrixes** [comment]
I agree, if nothing else because I can't find a way to refresh the feeds other than

1. Pull to refresh (which is really cumbersome if you've scrolled down far)
2. Actionbar -> Refresh option.

I'd really like to be able to either have a refresh button in the FAB or on a bottom tab.
Right now it's way too fiddly to refresh the feeds.

**Comment by Docile-Alligator** [comment]
You can customize the fab to act as a refresh button in settings.

**Comment by Tursko** [comment]
> You can customize the fab to act as a refresh button in settings.

Where can we do that? I know you can refresh by clicking the Infinity title, but that's not super intuitive.
